static/js/github-create-branch.js

```diff
@@ -1,11 +1,47 @@
+const createBranchPost = () => {
+	const url = "/github/create-branch";
+	// Todo improve this select2 get and split once we have real data coming in
+	const repoWithOwner = $("#ghRepo").select2('data').text.split(' / ');
```

Comment:
The repo name now uses `full_name`, so doesn't have the extra spaces anymore.
```suggestion
	const repoWithOwner = $("#ghRepo").select2('data').text.split('/');
```

Reply:
Oh nice good catch!!